labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -69,7 +90,10 @@ object RecursionPatternMatchingExercise {
    * List(List(1,2,3), List('A, 'B, 'C), List('a, 'b, 'c)) -> List(List(1, 'A, 'a), List(2, 'B, 'b), List(3, 'C, 'c))
    */
   def zipMultiple(in: List[List[_]]): List[List[_]] = {
-    error("fix me")
+    in match {
+      case (_ :: tail) :: others => in.map((l:List[_]) => l(0)) :: zipMultiple(in.map((l:List[_]) => l.tail))
+      case _ => List()
+    }
   }
```

Comment:
I never ended up casting something as an ambiguously typed list, but that's a great idea for this situation. It avoids a lot of annoying compiler errors and lets you do things like these efficient mapping functions. I'll try that in my future assignments.